services/spar/test/Test/Spar/APISpec.hs

```diff
@@ -26,4 +31,14 @@ spec = do
 
   it "roundtrip: IdPMetadataInfo" . property $ \(val :: IdPMetadataInfo) -> do
     let withoutRaw (IdPMetadataValue _ x) = x
-    (withoutRaw <$> (eitherDecode . encode) val) `shouldBe` Right (withoutRaw val)
+    (withoutRaw <$> (Aeson.eitherDecode . Aeson.encode) val) `shouldBe` Right (withoutRaw val)
+
+  describe "SSOSettings JSON instance" $ do
```

Comment:
Note that `validateEveryToJSON` doesn't roundtrip-test the aeson instances.  (But we should probably fix this there, and not start another list of types for which to roundtrip-test that we need to keep up to date manually.)